### PR TITLE
Feature/vizualizare liste

### PR DESCRIPTION
**Pull request by Lordsib**
Dumnezeu cu mila

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,228 @@
...
 <html lang="en">
 <head>
-    <title>HTMX-Demo</title>
+    <title>vizListe</title>
+    <script src="https://unpkg.com/htmx.org@1.6.1/dist/htmx.min.js"></script>
+    <script>
+        // id of the user. I have no fucking idea how I am supposed to get it
+        userId = 1
+        function loadCourses() {
+            fetch(`/${userId}/courses`)
```
Comment (Piciorus) [inline index.html:10]: add this in a separate js file and call it here

File: index.html [hunk 2]
```diff
@@ -1,11 +1,228 @@
...
+                .catch(error => console.error('Error fetching courses:', error));
+        }
+
+        document.addEventListener('DOMContentLoaded', loadCourses);
+    </script>
+    <style>
+        body {
+            background-color: #3d5a80;
+            color: white;
+            margin: 0;
```
Comment (Piciorus) [inline index.html:58]: add this css in a separate file

File: index.html [hunk 3]
```diff
@@ -1,11 +1,228 @@
...
+    </div>
+</div>
+<div class="overlay">
+    <div class="title-overlay">Cursurile mele</div>
+        <div class="course-container">
+            <div class="course-title">Obligatorii</div>
+            <div id="obligatoryCoursesContainer">
+            </div>
+                <div class="course-window">
+                    <div class="blue-ribbon"></div>
```
Comment (Piciorus) [inline index.html:202]: should this to be in another file html not in index?

File: src/main/java/org/example/Config/WebSecurityConfig.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package org.example.Config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@SuppressWarnings("deprecation")
```
Comment (Piciorus) [inline src/main/java/org/example/Config/WebSecurityConfig.java:10]: what is this?

**Review by Piciorus — CHANGES_REQUESTED** [review]

**Comment by Piciorus** [comment]
please reformat all files

**Comment by Piciorus** [comment]
please add a print screen to see how it looks like

File: index.html (hunk 3 above)
Comment (Lordsib) [reply index.html:202]: I don't think so. It's the same page

**Comment by Lordsib** [comment]
> please add a print screen to see how it looks like

[vizListe.pdf](https://github.com/732PC/ProiectColectiv732/files/13562594/vizListe.pdf)

**Pull request closed without merging**